sdk/samples/core/mosquitto_multiple_rpc_sample.c

```diff
@@ -500,6 +501,7 @@ int main(int argc, char* argv[])
   connection_options.username_buffer = username;
   connection_options.password_buffer = AZ_SPAN_EMPTY;
   connection_options.hostname = hostname;
+  connection_options.client_certificates = client_certificates;
```

Comment:
This line better be up above, for a better flow (more intuitive for user). Example:

```c
az_mqtt5_x509_client_certificate primary_credential = (az_mqtt5_x509_client_certificate){
    .cert = cert_path1,
    .key = key_path1,
  };
az_mqtt5_x509_client_certificate client_certificates[1] = { primary_credential };
...
connection_options.client_certificates = client_certificates;
```

Reply:
This looks easier to understand! Will make the changes